src/Compatibility/Core/src/WinUI/Resources.xaml

```diff
@@ -19,6 +19,8 @@
         <ResourceDictionary Source="PickerStyle.xaml" />
     </ResourceDictionary.MergedDictionaries>
 
+    <x:Boolean x:Key="MicrosoftMauiControlsCompatibilityIncluded">true</x:Boolean>
```

Comment:
Instead of looking for a random key, put a specific one in there for us to check and know for sure.